packages/node_modules/@webex/plugin-meetings/src/mediaQualityMetrics/config.js

```diff
@@ -193,7 +165,70 @@ export default {
                 ssci: 0// Not avaliable
               },
               h264CodecProfile: 'BP',
-              isActiveSpeaker: false, // Not avaliable
+              isActiveSpeaker: true,
+              optimalFrameSize: 0, // Not avaliable
```

Comment:
IMO we should indicate with -1 for those not available, and update with values for those that are.

Reply:
planning to remove it and - values causes errors on the result